stylesheets/pages/products.css

```diff
@@ -0,0 +1,63 @@
+@import url("../main.css");
+@import url("../layout/product-card.css");
+
+.page-main {
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  gap: 0 var(--spacing-sm);
+  width: 90vw;
+  margin: var(--spacing-md) auto;
+}
+/** filters **/
+.filters {
+  gap: var(--spacing-xs);
+  box-shadow: 1px 2px 10px rgb(238, 234, 234);
+  min-width: 17rem;
```

Comment:
you usually don't want to change the width of filters sidebar when screen size changes......so giving a max -width might be more relevant here in this case than giving a min-width or just add a max-width to this code too accordingly

Reply:
Yes, that makes sense to me. Changes done, please approve